### ISSUE TITLE
Calibration Factor Changes in TRME_RR

### ISSUE THREAD

**Issue by kkappler** [simpeg/aurora#27]
In TRME the variable cfac is defined as follows:
cfac = 1. / (2 * (1. - (1. + self.r0) * np.exp(-self.r0)))

In TRME_RR it is defined similarly but commented out Instead it is a simpler form:

```
 %cfac = 1./(2*(1.-(1.+ITER.r0)*exp(-ITER.r0) ));
if ITER.iterMax > 0
     notConverged = 1;
     cfac = 1./(1.-exp(-ITER.r0));
     %cfac = 1./(2*(1.-(1.+ITER.r0)*exp(-ITER.r0) ));
```

![cfac_compate_issue27](https://user-images.githubusercontent.com/1154939/134724572-3951d4cb-4b57-486b-877e-28a2b7ae0f3d.png)

Attached is a plot of the two expressions for cfac (aka 1/beta in Egbert & Booker 1986 (EB86)).  In EB86 beta is quoted as 0.7784 for r0=1.5.  Thus we can see that the simpler expression from TRME_RR is the better approximation.

**Comment by kkappler** [comment]
The cfac is .7784 for r0=1.5.  This is hard coded in the Fortran code.
cfac is the beta in Equation A3 in egbert and booker 1986.  
Note that A3 has a typo and the opening square bracket should be on the other side of the 1/sqrt(2pi)

The psi' FWIW comes from (A2)

**Comment by kkappler** [comment]
This can be made into a function or a look up table with interpolation.  The priority is currently low, but this issue is not closed.

TODOs: 
1. Compute beta for a few values of r0.  You can use r0=1.5-->beta=0.7784 as a test to sanity check the calculation
2. Either make a program that computes beta(r0) from analytic expression (using a Taylor expansion?) or one that interpolates.  In either case, cfac is still not properly integrated into the program.

**Comment by kkappler** [comment]
This issue was intended to resolve the problem of two definitions of "cfac", which we now know is 1/beta.
We have concluded that between the two expressions one is better:
```
cfac = 1./(1.-exp(-ITER.r0));
```
is the better choice for 1/beta.

A numeric solution following exactly Equation A3 in EB86 could be added, and I created issue #111 for this, but the issue with two calibration factors is closed.
